cibyl/models/ci/job.py

```diff
@@ -46,19 +46,29 @@ class Job(Model):
             'arguments': [Argument(name='--builds', arg_type=str,
                                    nargs="*", func="get_builds",
                                    description="Job builds")]
-        }
+        },
+        'last_build': {
```

Comment:
I'm wondering if "--last-build" can be just an additional argument
So:
```
 'arguments': [Argument(name='--builds', ...), Argument(name='--last-build')]
```
To get the last build, we can already take the last elements from builds attribute, so I would probably avoid having a separate attribute specifically for the last build.

Reply:
That's seems like a natural way to organize it, I completely missed the fact that you can specify more than one argument. I'll explore this way